### PR TITLE
Feature/jp 25 membership list

### PR DESCRIPTION
**Pull request by EinSev**

### CODE REVIEW

**Review by stefll — DISMISSED** [review]
Design Review: It looks very good!!

**Review by stefll — DISMISSED** [review]
Design Review: Looks very good !!

File: frontend/pages/journey/[id].vue [hunk 1]
```diff
@@ -47,6 +54,514 @@ if (error.value) {
...
+  "firstName": "Maddy",
+  "lastName": "Posse",
+  "role": 1
+}, {
+  "id": "Skilith",
+  "firstName": "Clyde",
+  "lastName": "Childes",
+  "role": 1
+}];
+
```
Comment (Mathematti) [inline frontend/pages/journey/[id].vue]: Remove test data and use actual data

File: frontend/pages/journey/[id].vue [hunk 2]
```diff
@@ -141,6 +152,17 @@ function copyToClipboard() {
           </button>
         </div>
       </div>
+      <div
+        class="flex flex-row justify-between items-center border-b pb-3 border-border-grey dark:border-input-placeholder">
+        <h1 class="text-xl text-footer dark:text-border-grey mt-2 -mb-2">
+          <T keyName="journey.sidebar.list.header" />
+        </h1>
+        <!-- <SvgEdit class="w-8" /> -->
```
Comment (Mathematti) [inline frontend/pages/journey/[id].vue:160]: Is this comment still needed

**Review by Mathematti — APPROVED** [review]
Code Review: lgtm

File: frontend/pages/journey/[id].vue (hunk 2 above)
Comment (EinSev) [reply frontend/pages/journey/[id].vue:160]: Ja für nächste US change roles

**Review by stefll — APPROVED** [review]

**Pull request merged**